sleap/config/frame_range_form.yaml

```
@@ -0,0 +1,13 @@
+
+  - name: min_frame_idx
+    label: Minimum frame index
+    type: int
+    range: 1,1000000
+    default: 1
+
+  - name: max_frame_idx
+    label: Maximum frame index
+    type: int
+    range: 1,1000000
+    default: 1000
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)